### PR TITLE
OCD-2838 UI Enhancements

### PR DESCRIPTION
**Pull request by mattystank**

### CODE REVIEW

File: src/app/components/listing/details/sed/edit/sed-task-add.jsx [hunk 1]
```diff
@@ -26,19 +27,29 @@ import { theme, utilStyles } from 'themes';
...
+    paddingBottom: '16px',
   },
   criteriaList: {
     display: 'flex',
     flexDirection: 'row',
   },
+  formContainer: {
+    display:'flex',
+    flexDirection:'column',
+    gap: 16,
```
Comment (andlar) [inline src/app/components/listing/details/sed/edit/sed-task-add.jsx:51]: I think this needs to be a string. I don't know if this is `16px` or `16em` or something else. With `0`, it's easy

File: src/app/components/listing/details/sed/edit/sed-task-edit.jsx [hunk 2]
```diff
@@ -154,7 +158,14 @@ function ChplSedTaskEdit({ task: initialTask }) {
         <CardContent>
           <Box className={classes.taskData}>
             <Card className={classes.fullWidthGridRow} id="summary">
-              <CardHeader title="Summary" />
+              <CardHeader
+                action={(
+                  <Button endIcon={<DeleteIcon fontSize="small" color="error" />} onClick={() => remove()} className={classes.deleteButton}>
```
Comment (andlar) [inline src/app/components/listing/details/sed/edit/sed-task-edit.jsx:136]: I like this in the header area, and it's good that it's "subtle", but I think we should put a red outline around it to help indicate it's a button

File: src/app/components/listing/details/sed/edit/sed-participants-edit.jsx [hunk 3]
```diff
@@ -145,78 +153,80 @@ function ChplSedParticipantsEdit({ task: initialTask }) {
...
+                endIcon={<AddIcon fontSize="medium" />}
+              >
+                Add Test Participant
+              </Button>
+            </Box>
+          )}
+        { addingParticipant
+          && (
+            <>
+              <CardContent>
```
Comment (andlar) [inline src/app/components/listing/details/sed/edit/sed-participants-edit.jsx:222]: What card is this inside?

Comment (andlar) [reply src/app/components/listing/details/sed/edit/sed-participants-edit.jsx:222]: Nevermind. I see what this is doing, and am okay with it, mostly.

File: src/app/components/listing/details/sed/edit/sed-task-edit.jsx (hunk 2 above)
Comment (mattystank) [reply src/app/components/listing/details/sed/edit/sed-task-edit.jsx:136]: please add to <Button **variant="outlined"**

File: src/app/components/listing/details/sed/edit/sed-task-add.jsx (hunk 1 above)
Comment (mattystank) [reply src/app/components/listing/details/sed/edit/sed-task-add.jsx:51]: 16px

**Pull request merged**